sdk/servicebus/service-bus/CHANGELOG.md

```diff
@@ -1,10 +1,11 @@
 # Release History
 
-## 1.1.3 (#Unreleased)
+## 1.1.3 (2020-02-10)
 
 - Fixes issue where the promise returned by `receiveMessages` would sometimes fail to settle when the underlying connection
   was disconnected while receiving messages.
-- Workaround for the [bug 6816](https://github.com/Azure/azure-sdk-for-js/issues/6816) affecting messages sent using the `scheduleMessage()` and `scheduleMessages()` methods has been added to the docs - [PR 6983](https://github.com/Azure/azure-sdk-for-js/pull/6983))
+  [PR 6601](https://github.com/Azure/azure-sdk-for-js/pull/6601)
+- Workaround for the [bug 6816](https://github.com/Azure/azure-sdk-for-js/issues/6816) affecting messages sent using the `scheduleMessage()` and `scheduleMessages()` methods has been added to the docs - [PR 6983](https://github.com/Azure/azure-sdk-for-js/pull/6983)
```

Comment:
Nit: I kind of read it better with an "A" at the beginning. Like "A workaround for the...", or perhaps moving the "added" to the beginning, "Added a workaround...". Take it or leave it! 🌞

Reply:
I agree! I was looking at the content for that (I didn't like the extra parenthesis) but glad to use better words :)